### PR TITLE
Documentation and examples for telegraf Source

### PR DESCRIPTION
**Pull request by nettorta**

### CODE REVIEW

File: docs/core_and_modules.rst [hunk 1]
```diff
@@ -1151,7 +1151,15 @@ List of metrics group names and particular metrics in them:
     * measure - default: call - metric value is a command or script execution output. Example: `<Custom measure="call" diff="1" label="Base size">du -s /var/lib/mysql/ | awk '{print $1}'</Custom>`
 * TelegrafRaw
     * raw telegraf TOML format, transparently added to final collector config 
-* Source additional source file in telegraf json format, can be used to add custom metrics that needs complex processing and do not fit into standart custom metrics (like log parsing with aggregation)
+* Source
+    additional source file in telegraf json format,     can be used to add custom metrics that needs complex processing and do not fit into standart custom metrics (like log parsing with aggregation)
```
Comment (direvius) [inline docs/core_and_modules.rst]: custom metrics do not include timestamps and this source does. This is the main difference. You can not import asynchronous data with custom metrics, but you can with this Source. Please, mention that.

File: docs/core_and_modules.rst [hunk 2]
```diff
@@ -1151,7 +1151,15 @@ List of metrics group names and particular metrics in them:
...
+
+    Config Host section example:
+    ``<Source>/path/to/file</Source>``
+
+    File format example:
```
Comment (direvius) [inline docs/core_and_modules.rst]: File format: jsonline. Each line is a json document. Example:

File: docs/core_and_modules.rst [hunk 3]
```diff
@@ -1151,7 +1151,15 @@ List of metrics group names and particular metrics in them:
...
+    ``{"fields":{"metric_name_1":0,"metric_name_2":98.27694231863998,},"name":"custom_group-name","tags":{"custom":"custom_tag","host":"hostname.tld"},"timestamp":1503990965}``
```
Comment (direvius) [inline docs/core_and_modules.rst]: what are tags for? It is not clear. And please mention that timestamp is obligatory.

File: docs/core_and_modules.rst (hunk 1 above)
Comment (direvius) [reply docs/core_and_modules.rst]: Also, did you forgot the '*' before the second line?

**Comment by direvius** [comment]
@nettorta please fix )

File: docs/core_and_modules.rst (hunk 3 above)
Comment (f2nd) [reply docs/core_and_modules.rst]: AFAIK we do not use tags in decoding at all.

**Comment by nettorta** [comment]
There you go

**Comment by direvius** [comment]
thanx

**Pull request merged**